algoliasearch-common/src/main/java/com/algolia/search/inputs/analytics/ABTest.java

```diff
@@ -0,0 +1,38 @@
+package com.algolia.search.inputs.analytics;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.List;
+
+public class ABTest {
```

Comment:
don't forget to add `Serializable` on all POJOs

Reply:
Oh right, good catch. Will do 👍